eng/common/scripts/Verify-Links.ps1

```diff
@@ -257,21 +272,29 @@ while ($pageUrisToCheck.Count -ne 0)
 
   $linkUris = GetLinks $pageUri
   Write-Host "Found $($linkUris.Count) links on page $pageUri";
-  
+  $badLinksPerPage = @{};
```

Comment:
I would just make this a list.

Reply:
Do we need to dedup links on the same page?

Reply:
You can still de-dup if it is a list you can still call Contains on it.